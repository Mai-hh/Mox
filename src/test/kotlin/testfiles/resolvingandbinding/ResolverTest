if (true) {
    var x = "hello";
    print x;
}

fun foo() {
    var x = "hello";
    print x;
}

foo();

fun foo2(x) {
    x = "hello";
    print x;
}

foo2("hi");

var x = 20;
fun updateX() {
    x = 30;
}

print x;
updateX();
print x;
