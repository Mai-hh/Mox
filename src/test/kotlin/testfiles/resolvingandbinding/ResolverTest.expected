hello
hello
hello
20
30